app/main.py

```diff
@@ -1,3 +1,66 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return "Distance(km=" + str(self.km) + ")"
+
+    def __add__(self, other: [int, float] | Distance) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: [int, float] | Distance) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: [int, float]) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: [int, float]) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: [int, float] | Distance) -> bool:
+        return (
+            self.km < other.km
+            if isinstance(other, Distance)
+            else self.km < other
+        )
+
+    def __gt__(self, other: [int, float] | Distance) -> bool:
+        return (
+            self.km > other.km
+            if isinstance(other, Distance)
+            else self.km > other
+        )
+
+    def __eq__(self, other: [int, float] | Distance) -> bool:
+        return (
+            self.km == other.km
+            if isinstance(other, Distance)
+            else self.km == other
+        )
+
+    def __le__(self, other: [int, float] | Distance) -> bool:
+        return (
+            self.km <= other.km
+            if isinstance(other, Distance)
+            else self.km <= other
+        )
+
+    def __ge__(self, other: [int, float] | Distance) -> bool:
+        return (
+            self.km >= other.km
+            if isinstance(other, Distance)
+            else self.km >= other
+        )
```

Comment:
you can write these two methods in one line each. Use previously created magic methods

Reply:
Nice idea, actually. thnx. I ddnt know we can do it that way